tests/main/interfaces-gpio-memory-control/task.yaml

```diff
@@ -0,0 +1,57 @@
+summary: Ensure that the gpio physical memory control interface works.
+
+details: |
+    The gpio-memory-control interface allows read/write access to all gpio memory.
+
+systems: [ubuntu-core-16-arm-*]
+
+environment:
+    CONNECTED_PATTERN: ':gpio-memory-control +test-snapd-gpio-memory-control'
+    DISCONNECTED_PATTERN: '\- +test-snapd-gpio-memory-control:gpio-memory-control'
+
+prepare: |
+    echo "Given the test-snapd-gpio-memory-control snap is installed"
+    snap install test-snapd-gpio-memory-control
+
+restore: |
+    rm -f call.error
+
+execute: |
+    if ! [ -c /dev/gpiomem ]; then
+        echo "The /dev/gpiomem device does not exist in current system"
+        exit 0
+    fi
+
+    echo "The interface is not connected by default"
+    snap interfaces | MATCH "$DISCONNECTED_PATTERN"
+
+    echo "When the interface is connected"
+    snap connect test-snapd-gpio-memory-control:gpio-memory-control
+    snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    mem_address=$(journalctl | grep -E 'gpiomem.*Registers at' | sed 's/^.*at //')
+    if [ -n $mem_address ]; then
+        echo "Memory address not registered"
+        exit 1
+    fimem_address=$(cat /var/log/syslog | grep -m1 'gpiomem.*Registers at' | sed 's/^.*at //')
```

Comment:
Missing newline after `fi`.

Reply:
Not sure I follow the piece here. If there's no long in the journal we'll exit the test. The second read from syslog will not happen at all. Perhaps this was meant to be a fallback? As in try journalctl, if that failed try syslog, if that failed too then exit.

Reply:
I updated that, journalctl cannot be used anymore because we clean it during the reset. 
I made a grep in the py and the only place I found this memory address is in the logs. I already checked on /sys and /proc but nothing.